source/component/PasDoc_Items.pas

```diff
@@ -3058,7 +3059,8 @@ function CioTypeToString(const CioType: TCIOType): String;
     'object',
     'packed object',
     'record',
-    'packed record'
+    'packed record',
+    'type' {helper only}
```

Comment:
Let's make it `'type helper'`, not just `'type'`. 

Similar to previous comment, this makes it more obvious, and then there's no need for a comment `{helper only}` here.

Reply:
It is kind of the same remark here. "record" is the label used for "record helper". If we add "helper" here, then the helper label would become "type helper helper".